vignettes/getting-started.Rmd

```diff
@@ -16,15 +16,27 @@ knitr::opts_chunk$set(
 
 # gimap tutorial
 
+TODO: Describe what this genetic interaction analysis actually is and why someone would want to do it. 
+
```

Comment:
```suggestion
## TODO: Describe what this genetic interaction analysis actually is and why someone would want to do it. **still a work in progress** I think I am still missing "what is it actually doing"

gimap performs analysis of dual-targeting CRISPR screening data, with the goal of aiding the identification of genetic interactions (e.g. cooperativity, synthetic lethality) in models of disease and other biological contexts. gimap analyzes functional genomic data generated by the pgPEN (paired guide RNAs for genetic interaction mapping) approach, quantifying growth effects of single and paired gene knockouts upon application of CRISPR library.
```

Reply:
This is great! links to review articles or easy to read scientific blogs are always great too!